src/services/mongodb/index.ts

```diff
@@ -45,8 +43,7 @@ export const setupMongoConnection = async () => {
     })
   } catch (err) {
     baseLogger.fatal({ err, user, address, db }, `error connecting to mongodb`)
-    await sleep(100)
-    exit(99)
+    throw err
```

Comment:
@bodymindarts @krtk6160 dont we need the exit code to handle CI errors??

Reply:
I don't think so. I was the one introducing this logic. but it's not very practical. node would still exit with 1 if an error is being thrown AFAIK